src/app/account.service.ts

```diff
@@ -74,6 +82,42 @@ export class AccountService {
     }
   }
 
+  getDerivedUser(publicKey: string, blockHeight: number): DerivedUserInfo{
```

Comment:
nit: Please rename to getDerivedPrivateUser, since this returns the seed hex for the derived key.